client/apollo/js/View/Track/SequenceTrack.js

```diff
@@ -937,11 +937,16 @@ var SequenceTrack = declare( "SequenceTrack", DraggableFeatureTrack,
         else {
             var plusDiv = dojo.create("div", { }, content);
             var minusDiv = dojo.create("div", { }, content);
+	    var comment = dojo.create("div", { }, content);
```

Comment:
@cmdcolin  It looks like this is the SequenceAlteration code.  It should probably be reverted as I think that #790 takes care of this.